package.json

```diff
@@ -27,7 +27,8 @@
     "precompile": "gts clean"
   },
   "dependencies": {
-    "google-gax": "^4.0.3"
+    "google-gax": "^4.3.0",
```

Comment:
Looks like you're updating the auth too, but you might just want to make these `^4.3.1` to be safe.

Reply:
bumped the version